test/SummaryClient.spec.ts

```diff
@@ -59,8 +62,52 @@ describe('Xray summary tests', () => {
         expect(secondIssue.severity).toBe('High');
     });
 
+    test('Artifact Summary component CVE', async () => {
+        const response: ISummaryResponse = await xrayClient.summary().component(express);
+        expect(response).toBeTruthy();
+
+        const issues: IIssue[] = response.artifacts[0].issues;
+        expect(issues.length).toBeGreaterThanOrEqual(1);
+
+        const testIssue: IIssue | undefined = issues.find((issue) => issue.summary === EXPRESS_ISSUE_SUMMARY);
+        expect(testIssue).toBeTruthy();
+
+        const cves: ICve[] | undefined = testIssue?.cves;
+        expect(cves).toBeTruthy();
+        if (!cves) {
+            return;
+        }
+        expect(cves).toHaveLength(1);
+        expect(cves[0].cve).toBe('CVE-2014-6393');
+        expect(cves[0].cvss_v2).toBe('4.3/CVSS:2.0/AV:N/AC:M/Au:N/C:N/I:P/A:N');
+    });
+
+    test('Artifact Summary component fixed versions', async () => {
+        const response: ISummaryResponse = await xrayClient.summary().component(express);
+        expect(response).toBeTruthy();
+
+        const issues: IIssue[] = response.artifacts[0].issues;
+        expect(issues.length).toBeGreaterThanOrEqual(1);
+
+        const testIssue: IIssue | undefined = issues.find((issue) => issue.summary === EXPRESS_ISSUE_SUMMARY);
+        expect(testIssue).toBeTruthy();
+
+        const components: IVulnerableComponent[] | undefined = testIssue?.components;
+        expect(components).toBeTruthy();
+        if (!components) {
```

Comment:
What is the reason we return here? Shouldn't this fail the test?

Reply:
Without it I got the following error in the test:
`test/SummaryClient.spec.ts:98:68 - error TS2532: Object is possibly 'undefined'.`
I change it a bit to improve readability.